modules/channel-twilio/src/backend/client.ts

```diff
@@ -149,7 +149,7 @@ export class TwilioClient {
         }
       }
 
-      const args = { mediaUrl: picture }
+      const args = { mediaUrl: picture ? picture : undefined }
```

Comment:
I don't understand, you say that when picture is undefined an error is thrown and to fix this you replace picture by undefined ? That will still throw an error ..

Reply:
Yes my bad, I shouldn't have said "when the mediaUrl is not defined" what i wanted to say is when the picture field is not fulfilled, the module will receive a null object and throw this error 

`Unhandled Rejection [Error, Media urls:  are invalid. Please use only valid http and https urls]`

Reply:
oh, ok.  I understand